roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -37,6 +37,40 @@ spec:
 {% endfor %}
 {% endif %}
       containers:
+        - name: proxy
+          image: {{openshift_logging_elasticsearch_proxy_image_prefix}}:{{openshift_logging_elasticsearch_proxy_image_version}}
+          imagePullPolicy: Always
+          args:
+           - --upstream-ca=/etc/elasticsearch/secret/admin-ca
+           - --https-address=:4443
+           - -provider=openshift
+           - -client-id={{openshift_logging_elasticsearch_prometheus_user}}
+           - -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
+           - -cookie-secret={{ 16 | oo_random_word | b64encode }}
+           - -upstream=https://localhost:9200
+           - '-openshift-sar={"namespace": "{{ openshift_logging_elasticsearch_namespace}}", "verb": "get", "resource": "logging-metrics"}'
```

Comment:
Where is `openshift_logging_elasticsearch_namespace` configured?  I would prefer if we picked something like `openshift-metrics` or `openshift-logging`.  If we really want to give people the choice (I do not want to), it must be prefixed with `openshift-`

Change the SAR to:

`{"namespace": "{{ openshift_logging_elasticsearch_namespace}}", "verb": "view", "resource": "prometheus", "group": "metrics.openshift.io"}`

Reply:
It is one of the defaults and is historical.  @smarterclayton mentioned that in another PR but logging lives in the 'logging' namespace for reasons never resolved.  This will take more work to change unless the namespace of the SAR is irrespective of where the proxy lives.

Reply:
Meh, I hate legacy stuff.  But OK.